docs/core/build/distribution-packaging.md

```diff
@@ -171,6 +171,8 @@ Most distributions require all artifacts to be built from source. This has some
 
 Multiple `dotnet-sdk` packages may provide the same files for the `NuGetFallbackFolder`. To avoid issues with the package manager, these files should be identical (checksum, modification date, and so on).
 
+The folders marked with `(*)` are used by multiple packages. Some package formats (e.g. `rpm`) require special handling of such folders. The package maintainer must take care of this.
```

Comment:
It would be nice to have this described somewhere... but this note LGTM--hard to make it clear to readers not already familiar with this requirement without putting a lot more info in this doc and probably cluttering it up. Selfishly, I'd like to get the Microsoft RPMs up to snuff, and know what the consequences are of getting it wrong.

Reply:
Yes, we can assume that the package maintainer knows what this is about.